docs/RELEASING.md

```diff
@@ -49,7 +55,15 @@
 
 ### Create release on F-Droid
 
-TODO
+1. Fetch the latest changes from the *fdroiddata* repository.
+2. Switch to a new branch in your copy of the *fdroiddata* repository.
+3. Edit `metadata/com.fsck.k9.yml` to create a new entry for the version you want to release. Usually it's copy & paste of the previous entry and adjusting `versionName`, `versionCode`, and `commit` (use the tag name). Leave `CurrentVersion` and `CurrentVersionCode` unchanged. Those specify which version is the stable/recommended build.
+4. Commit the changes. Message: "Update K-9 Mail to $newVersionName"
+5. Run `$pathToFdroidserver/fdroid build --latest com.fsck.k9` to build the project using F-Droid's toolchain. 
```

Comment:
This works only with a cloned `frdoidserver`, in case the tooling was installed: `fdroid build --latest com.fsck.k9` is enough.

Reply:
Updated by a32473af457697fcec35fdfbcba07af8a8107f3e